test/e2e/proxy_test.go

```diff
@@ -736,11 +736,11 @@ func setStoneSoupConfig(t *testing.T, hostAwait *wait.HostAwaitility, memberAwai
 func verifyHasExpectedWorkspace(t *testing.T, expectedWorkspace toolchainv1alpha1.Workspace, actualWorkspaces ...toolchainv1alpha1.Workspace) {
 	for _, actualWorkspace := range actualWorkspaces {
 		if actualWorkspace.Name == expectedWorkspace.Name {
-			assert.Equal(t, expectedWorkspace.Status, actualWorkspace.Status)
-			assert.Equal(t, expectedWorkspace.ObjectMeta.ResourceVersion, actualWorkspace.ObjectMeta.ResourceVersion)
-			assert.Equal(t, expectedWorkspace.ObjectMeta.Generation, actualWorkspace.ObjectMeta.Generation)
-			assert.Equal(t, expectedWorkspace.ObjectMeta.CreationTimestamp, actualWorkspace.ObjectMeta.CreationTimestamp)
-			assert.Equal(t, expectedWorkspace.ObjectMeta.UID, actualWorkspace.ObjectMeta.UID)
+			assert.NotEmpty(t, actualWorkspace.Status, "Workspace.Status field is empty", actualWorkspace, expectedWorkspace)
+			assert.NotEmpty(t, actualWorkspace.ObjectMeta.ResourceVersion, "Workspace.ObjectMeta.ResourceVersion field is empty: %#v", actualWorkspace)
```

Comment:
In these cases the actual value would be empty if it fails so there's not much value in having an explicit string for the failure case. Can simplify it to:
```suggestion
			assert.NotEmpty(t, actualWorkspace.ObjectMeta.ResourceVersion)
```

Same for the other checks below

Reply:
I'm printing to whole `actualWorkspace` object in case of an assertion error, so that all fields and values can be checked. 
But if it's not useful I can remove that.

Reply:
I personally found it useful to see all the values in case of an assertion failure , and not just the generic error about the field that was empty, with the line number. But we can remove those at any point if needed.